exercicios/para-casa/sistema_de_alunas_patricia.py

```diff
@@ -0,0 +1,134 @@
+#from dataset_alunas import dataset
+dataset = []
```

Comment:
O código não está utilizando o dataset proposto no exercício.

**Sugestão:**
```suggestion
from dataset_alunas import dataset
```